File: LPS_Service/Models/Customer/CustomerInfoModel.cs
using LPS_Service.Entity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace LPS_Service.Models.Customer
{
    public class CustomerInfoModel
    {
        public SpCustomerModel customerInfo { get; set; }
        public CustomerGroup[] customerGroup { get; set; }
        public SpCustomerRating[] creditRating { get; set; }
        public CustomerContact[] contactInfo { get; set; }
        public CustomerWatchListModel[] watchlist { get; set; }
    }
    public class CustomerGroup
    {
        public string groupNo { get; set; }
        public string groupName { get; set; }
        public SpCustomerGroupModel[] data { get; set; }
    }
    public class CustomerContact
    {
        public string CONTACT_NAME { get; set; }
        public string CONTACT_TEL { get; set; }
    }
}

File: LPS_Angular/src/app/models/Customer/CustomerInfoModel.ts
import { CustomerModel } from './CustomerModel';

export class CustomerInfoModel {
    // cif: string;
    // registerDate: string;
    // address: string;
    // telephone: string;
    // fax: string;
    // noStaff: number;
    // netAsset: number;
    // netProfit: number;
    // businessSize: string;
    // classification: string;
    // watchList: string;
    // reschedule: string;
    customerInfo: CustomerModel;
    customerGroup: CustomerGroup[];
    creditRating: CreditRating[];
    contactInfo: ContactInfo[];
    watchlist: CustomerWatchList[];
}
class ContactInfo {
    CONTACT_NAME: string;
    CONTACT_TEL: string;
}
export class CustomerGroup {
    groupNo: number;
    groupName: string;
    data: CustomerGroupData[];
}
class CustomerGroupData {
    cif: string;
    customerName: string;
    relation: string;
}
class CreditRating {
    RATING_TYPE: string;
    RATING_COMPANY: string;
    RATING: string;
    RATING_DATE: string;
}
class CustomerWatchList {
    WATCH_LIST_FLAG: string;
    WATCH_LIST_NOTE: string;
    CREATE_DATE: string;
    MATURITY_DATE: string;
}
